hnn_core/cell.py

```diff
@@ -67,6 +67,9 @@ class _Cell(ABC):
     ----------
     pos : list of length 3
         The position of the cell.
+    rec_v : h.Vector()
```

Comment:
why is it called `rec_v` when the parameter is called `record_vsoma` ? I think `vsoma` makes more sense as a name?

Reply:
This follows the naming scheme above with `rec_i` for currents. I don't think it should be called `vsoma` because this is the only place the voltage is stored as a hoc object. `vsoma` is a python dictionary everywhere else.